src/resdk/resolwe.py

```diff
@@ -462,7 +462,11 @@ def get_or_run(self, slug=None, input={}):
         return Data(resolwe=self, **model_data)
 
     def _download_files(
-        self, files: List[Union[str, Path]], download_dir=None, show_progress=True
+        self,
+        files: List[Union[str, Path]],
+        download_dir=None,
+        show_progress=True,
+        custom_file_names: Union[List[str], None] = None,
```

Comment:
It would be better to make a mapping of field names to custom file names. If this would be suitable, I can make the change. The problem is that you do not know the file names in advance and this would especially prove to be difficult when downloading data for multiple samples.